components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -162,8 +162,8 @@ func UnmanagedArgoWorkFlowExists(ctx context.Context,
 	// Verify if existing workflow is deployed by ODH with label
 	odhLabelValue, odhLabelExists := workflowCRD.Labels[labels.ODH.Component(ComponentName)]
 	if odhLabelExists && odhLabelValue != "true" {
-		return fmt.Errorf(" %v CRD already exists but not deployed by this operator. Remove existing Argo workflows or set datasciencepipelines to Removed to proceed ",
-			ArgoWorkflowCRD)
+		return fmt.Errorf(" %v CRD already exists but not deployed by this operator. "+
+			"Remove existing Argo workflows or set `spec.components.datasciencepipelines.managementState` to Removed to proceed ", ArgoWorkflowCRD)
```

Comment:
```suggestion
		return fmt.Errorf("%v CRD already exists but is not managed by this operator. "+
			"Remove existing Argo workflows or set data science pipelines  management state to Removed to proceed ", ArgoWorkflowCRD)
```

What does the `%v` prints? The whole struct? Shouldn't we print GVK instead?

Reply:
ArgoWorkflowCRD is set to string  "workflows.argoproj.io"
%s is enough here than %v

as for the "data science pipelines management state", was from https://github.com/opendatahub-io/opendatahub-operator/pull/967#discussion_r1565927271